code/obj/item/cigarette.dm

```diff
@@ -258,7 +258,10 @@
 
 			var/mob/living/carbon/human/human_target = target
 			if (human_target && rand(1,5) == 1)
-				SPAWN(0) target.emote("cough")
+				SPAWN(0)
+				target.emote("cough")
+				if (rand(1, 5) == 1)
```

Comment:
BYOND has a quicker wait to do this kind of thing with the prob proc, i.e. prob(20) for a 20% chance